ui-admin/src/study/notifications/TriggerDesignerEditor.tsx

```diff
@@ -452,12 +460,32 @@ const TaskStatusEditor = (
       />
     </div>
     <div>
-      <label className="form-label mt-3" htmlFor="updateTaskTargetStableId">Target stable id </label>
+      <label className="form-label mt-3" htmlFor="updateTaskTargetStableIds">Target stable id </label>
       <InfoPopup content={<span>
             the stable id of the task to update. For survey tasks, this is the survey stable id.
       </span>}/>
-      <TextInput value={trigger.updateTaskTargetStableId} id="updateTaskTargetStableId"
-        onChange={v => updateTrigger('updateTaskTargetStableId', v)}/>
     </div>
   </div>
 }
+
+const TaskTargetStableIdsEditor = ({ studyEnvParams, stableIds, setStableIds }:
+  {studyEnvParams: StudyEnvParams, stableIds: string[], setStableIds: (ids: string[]) => void}) => {
+  const [surveys, setSurveys] = useState<Survey[]>([])
+  useLoadingEffect(async () => {
+    const studyEnvSurveys = await Api.findConfiguredSurveys(studyEnvParams.portalShortcode,
+      studyEnvParams.studyShortcode, studyEnvParams.envName, true)
+    setSurveys(studyEnvSurveys.map(studyEnvSurvey => studyEnvSurvey.survey))
+  })
+  stableIds = stableIds ?? []
+  const inputId = useId()
+  return <div className="mt-3">
+    <label className="form-label" htmlFor={inputId}>Limit to these surveys
+      <span className="fst-italic ms-2">(leave blank if trigger applies to all)</span></label>
+    <Select options={surveys} inputId={inputId}
+      value={stableIds.map(stableId => surveys.find(survey => survey.stableId === stableId))}
```

Comment:
this doesn't seem to display correctly for the new med hx complete trigger:
<img width="1244" alt="image" src="https://github.com/user-attachments/assets/d4941851-c55c-4bf7-94b0-9dc4e6fcdceb" />

in addition, when I try to use this to edit the exisitng med hx complete trigger to change the survey filters, it doesn't seem to work- the database shows a new trigger with just the med hx survey stable id again.